client/store.js

```diff
@@ -782,6 +791,16 @@ export let Actions = {
             accessId: account.bankAccess
         });
     },
+    
+    SetOperationType(operation, typeId) {
+        assert(operation instanceof Operation, 'SetOperationType expects an Operation as the first argument');
+        assert(typeof typeId === 'string', 'SetOperationType expects a String category id as the second argument');
+        flux.dispatch({
+            type: Events.user.updated_type_of_operation,
+            operation,
+            typeId: typeId
```

Comment:
line 801? If you loog at line 811, the ',' is not here. Do I update all the others?